exercise.schema.json

```diff
@@ -75,12 +75,51 @@
           "description": "List of answer fields which can be used within this question",
           "type": "array",
           "items": {"$ref": "#/definitions/answer_field"}
+        },
+        "settings": {
+          "description": "Settings which influence the way the question is to be consumed by the player",
+          "$ref": "#/definitions/question_settings"
         }
       },
       "required": [
         "id", "type", "question", "answers"
       ]
     },
+    "question_settings" : {
+      "type": "object",
+      "properties": {
+        "max_attempts": {
+          "description": "The number of attempts a user has for answering the question correctly",
+          "type": "number",
+          "default": 3
+        },
+        "attempt_settings": {
+          "description": "Settings which apply to a specific attempt. The index of the array is used as the number of the attempt the setting applies to. If no specific settings are given the value will be the default of the property",
+          "type": "array",
```

Comment:
@Graspleric i don't think this is an array. The index is important and does not start at zero. It starts at 1 and in theory you could have an entry like:
```
attempt_settings: {
 1:  { "percentage_of_full_points": 100},
 3:  { "percentage_of_full_points": 25},
}
```